### PR TITLE
Split assets data to better reflect year dependency

### PR DESCRIPTION
**Pull request by abelsiqueira**
Update assets data to reflect year dependency better

### CODE REVIEW

File: src/objective.jl [hunk 1]
```diff
@@ -112,26 +112,28 @@ function add_objective!(model, graph, dataframes, representative_periods, sets,
...
 
+        # FIXME: Ugly hack applied
         flows_variable_cost = @expression(
             model,
             sum(
                 weight_for_operation_discounts[row.year] *
                 representative_periods[row.year][row.rep_period].weight *
                 duration(row.timesteps_block, row.rep_period, representative_periods[row.year]) *
-                graph[row.from, row.to].variable_cost[row.year] *
+                (graph[row.from, row.to].variable_cost[row.year] |> values |> first) *
```
Comment (abelsiqueira) [inline src/objective.jl:122]: Here

File: src/objective.jl [hunk 2]
```diff
@@ -112,26 +112,28 @@ function add_objective!(model, graph, dataframes, representative_periods, sets,
...
 
+        # FIXME: Ugly hack applied
         units_on_cost = @expression(
             model,
             sum(
                 weight_for_operation_discounts[row.year] *
                 representative_periods[row.year][row.rep_period].weight *
                 duration(row.timesteps_block, row.rep_period, representative_periods[row.year]) *
-                graph[row.asset].units_on_cost[row.year] *
+                graph[row.asset].units_on_cost[row.year][row.year] *
```
Comment (abelsiqueira) [inline src/objective.jl:134]: And here

**Review by datejada — APPROVED** [review]
LGTM. The "XXX-both.csv" files should be renamed. @gnawin and I can create a better name during the week and change them accordingly.

**Comment by abelsiqueira** [comment]
Thanks for the review

**Pull request merged**